nipoppy/cli/parser.py

```diff
@@ -148,6 +148,17 @@ def add_arg_verbosity(parser: _ActionsContainer) -> _ActionsContainer:
     return parser
 
 
+def add_arg_keepworkdir(parser: _ActionsContainer) -> _ActionsContainer:
+    """Add a --keep_workdir argument to the parser."""
+    parser.add_argument(
+        "--keep_workdir",
```

Comment:
Use kebab-style to be consistent with other CLI args
```suggestion
        "--keep-workdir",
```

Also should update docstring after this change is made